application/views/home.php

```diff
@@ -8,7 +8,19 @@
 </div>
 
 <div id="bot_summary">
-	{BotPieceSummary} 
+		<table>
```

Comment:
Also...let's add a `<caption>` line right after `<table>` for a table caption. (suggestion)